platform-hub-web/src/app/identities/identities-manager.component.js

```diff
@@ -0,0 +1,54 @@
+export const IdentitiesManagerComponent = {
+  template: require('./identities-manager.html'),
+  controller: IdentitiesManagerController
+};
+
+function IdentitiesManagerController($scope, hubApiService, gitHubIdentityService, events, _) {
+  'ngInject';
+
+  const ctrl = this;
+
+  ctrl.supported = [
+    {provider: 'keycloak', title: 'Keycloak (using Office 365)', external: false},
+    {provider: 'github', title: 'GitHub', external: true}
+  ];
+
+  ctrl.identities = {};
```

Comment:
I can't do that in this particular case because `ctrl` is just a reference to `this` and I want to bind these things to the context provided by `this`.